azure-pipelines.yml

```diff
@@ -17,10 +17,10 @@ jobs:
     strategy:
       matrix:
```

Comment:
@jhendrixMSFT @ArcturusZhang do you know who owns the track 1 modules that might be able to sign-off on this change?